zerver/webhooks/taiga/view.py

```diff
@@ -242,61 +261,64 @@ def parse_change_event(change_type: str, message: Mapping[str, Any]) -> Optional
         # we are not supporting this type of event
         return None
 
-    evt.update({"type": message["type"], "event": event_type, "values": values})
+    evt.update({'type': message['type'], 'event': event_type, 'values': values})
     return evt
 
 def parse_webhook_test(message: Mapping[str, Any]) -> Dict[str, Any]:
     return {
-        "type": "webhook_test",
-        "event": "test",
-        "values": {
-            "user": get_owner_name(message),
-            "end_type": "test"
+        'type': 'webhook_test',
+        'event': 'test',
+        'values': {
+            'user': get_owner_name(message),
+            'user_link': get_owner_link(message),
+            'end_type': 'test'
         }
     }
 
 
 def parse_message(message: Mapping[str, Any]) -> List[Dict[str, Any]]:
-    """ Parses the payload by delegating to specialized functions. """
+    ''' Parses the payload by delegating to specialized functions. '''
     events = []
-    if message["action"] in ['create', 'delete']:
+    if message['action'] in ['create', 'delete']:
         events.append(parse_create_or_delete(message))
-    elif message["action"] == 'change':
-        if message["change"]["diff"]:
-            for value in message["change"]["diff"]:
+    elif message['action'] == 'change':
+        if message['change']['diff']:
+            for value in message['change']['diff']:
                 parsed_event = parse_change_event(value, message)
                 if parsed_event:
                     events.append(parsed_event)
-        if message["change"]["comment"]:
+        if message['change']['comment']:
             events.append(parse_comment(message))
-    elif message["action"] == "test":
+    elif message['action'] == 'test':
         events.append(parse_webhook_test(message))
```

Comment:
It looks like you wouldn't have needed to change this function at all were you not reworking the quote formatting.